### DISCUSSION TITLE
Error: "The `content` option in your Tailwind CSS configuration is missing or empty."

### DISCUSSION THREAD

**Discussion by roomman** [tailwindlabs/tailwindcss#7960] [Help]
**What version of Tailwind CSS are you using?**

v3.0.23

**What build tool (or framework if it abstracts the build tool) are you using?**

Tailwind CLI

**What version of Node.js are you using?**

v16.13.2

**What browser are you using?**

Safari

**What operating system are you using?**

macOS

**Describe your issue**

The following error is shown in my terminal when I launch Tailwind via the CLI:
```
warn - The `content` option in your Tailwind CSS configuration is missing or empty.
warn - Configure your content sources or your generated CSS will be missing styles.
warn - https://tailwindcss.com/docs/content-configuration
```
My config file has `content`:
```
const { join } = require('path');

module.exports = {
  content: [
    join(__dirname, './app', 'index.html'),
    join(__dirname, './app', '**', '*.hbs'),
    join(__dirname, './app', '**', '*.js'),
    join(__dirname, 'node_modules', 'ember-power-select', 'addon', 'components', '**', '*.hbs'),
  ],
...
```
The config file is stored in the root of my project. All styles are being resolved correctly.

**Comment by RobinMalfait** [comment]
Hmm that sounds odd, can you provide a minimal reproduction repository so that we can take a look?

**Comment by daskyrk** [comment]
Maybe the reason is both `content` and `purge` property exist in your config file.
